client/lib/purchases/index.js

```diff
@@ -85,14 +85,21 @@ function getPartnerName( purchase ) {
 	return null;
 }
 
+// TODO: refactor to avoid returning a localized string.
```

Comment:
@jsnajdr: I intend to fix this, but I'm not sure if it falls within the scope of this PR, or whether it's best handled in a separate one. What do you think?